homeassistant/components/sensor/hive.py

```diff
@@ -0,0 +1,746 @@
+"""Hive Integration - sensor."""
+import logging
+from homeassistant.const import TEMP_CELSIUS
+from homeassistant.components.sensor import ENTITY_ID_FORMAT
+from homeassistant.helpers.entity import Entity
+from homeassistant.loader import get_component
+
+DEPENDENCIES = ['hive']
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def setup_platform(hass, config, add_devices,
+                   device_list, discovery_info=None):
+    """Setup Hive sensor devices."""
+    hive_comp = get_component('hive')
+
+    if len(device_list) > 0:

```

Comment:
Just a nitpick, but the outer if isn't necessary here. If the length is zero, the loop won't iterate.

Reply:
outer removed, thanks for pointing it out, have been meaning to remove this for a while.